doc-content/enterprise-only/openshift/rhpam72-prod-ref.adoc

```diff
@@ -484,7 +495,7 @@ information.
 |`SSO_DISABLE_SSL_CERTIFICATE_VALIDATION` | RH-SSO Disable SSL Certificate Validation | `${SSO_DISABLE_SSL_CERTIFICATE_VALIDATION}`
 |`SSO_PRINCIPAL_ATTRIBUTE` | RH-SSO Principal Attribute to use as username. | `${SSO_PRINCIPAL_ATTRIBUTE}`
 |`HOSTNAME_HTTP` | Custom hostname for http service route.  Leave blank for default hostname, e.g. <application-name>-smartrouter-<project>.<default-domain-suffix>' | `${KIE_SERVER1_HOSTNAME_HTTP}`
-|`HOSTNAME_HTTPS` | Custom hostname for https service route.  Leave blank for default hostname, e.g.: secure-<application-name>-rhpamcentrmon-<project>.<default-domain-suffix> | `${KIE_SERVER1_HOSTNAME_HTTPS}`
```

Comment:
similar, not sure what this should be, but I don't think it should be smartrouter

Reply:
Again I can't make this change